lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +139,15 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($type === \PDO::PARAM_LOB || -99 === $type) {
```

Comment:
I don't think we can add another constant for CLOB without breaking BC (because we would have to change the binding types in DBAL's Type classes). Can't we maybe detect whether we need `\PDO::PARAM_STR` or `\OCI_TEMP_CLOB` according to the given value? Then we could get rid of the pseudo binding type `-99` and don't need to introduce a new type constant.